content/en/serverless/installation/dotnet.md

```diff
@@ -36,11 +36,15 @@ You need to subscribe the Datadog Forwarder Lambda function to each of your func
 1. [Install the Datadog Forwarder if you haven't][2].
 2. [Subscribe the Datadog Forwarder to your function's log groups][5].
 
+### Unified Service Tagging
```

Comment:
Do we need a section heading here? If it's just one section, we can leave this as a note or bold it

Reply:
It's only a subsection under the Configuration section, but I can go either way.